api/disgo.go

```diff
@@ -52,6 +52,9 @@ type Disgo interface {
 	GetGuildCommandPermissions(guildID Snowflake, commandID Snowflake) (*GuildCommandPermissions, restclient.RestError)
 	SetGuildCommandsPermissions(guildID Snowflake, commandPermissions ...SetGuildCommandPermissions) ([]*GuildCommandPermissions, restclient.RestError)
 	SetGuildCommandPermissions(guildID Snowflake, commandID Snowflake, permissions SetGuildCommandPermissions) (*GuildCommandPermissions, restclient.RestError)
+
+	GetTemplate(code string) (*GuildTemplate, restclient.RestError)
+	CreateGuildFromTemplate(templateCode string, data CreateGuildFromTemplate) (*Guild, restclient.RestError)
```

Comment:
```suggestion
	CreateGuildFromTemplate(templateCode string, createGuildFromTemplate CreateGuildFromTemplate) (*Guild, restclient.RestError)
```

Reply:
this one & a few others are still missing 🤔

Reply:
it doesn't appear to be missing on the latest commits?